steps/start/independence/content.cy.json

```diff
@@ -3,5 +3,6 @@
   "title": "Tribiwnlys annibynnol fydd yn penderfynu ar eich apêl",
   "reviewed": "Bydd eich apêl yn cael ei hadolygu gan banel tribiwnlys sy’n cynnwys {{tribunalPanel}}. Byddant yn ystyried popeth y byddwch yn ei ddweud wrthynt ac yn penderfynu ar eich hawl i gael {{benefitEndText}}{{benefitType}}.",
   "separate": "Mae'r tribiwnlys ar wahân i'r Adran Gwaith a Phensiynau.",
-  "UCreviewed": "Caiff eich apêl ei hadolygu gan dribiwnlys sy’n cynnwys barnwr ac, ar gyfer rhai apeliadau, meddyg. Byddant yn ystyried popeth rydych chi’n ei ddweud wrthynt ac yn gwneud penderfyniad."
+  "UCreviewed": "Caiff eich apêl ei hadolygu gan dribiwnlys sy’n cynnwys barnwr ac, ar gyfer rhai apeliadau, meddyg. Byddant yn ystyried popeth rydych chi’n ei ddweud wrthynt ac yn gwneud penderfyniad.",
```

Comment:
For the Welsh content that we don't have, I would use the English IBA stuff rather than the old Welsh stuff. If we don't get time and the Welsh doesn't get done, then it will be giving false information, I think it's better to be correct info but wrong language than the correct language but wrong info (this applies across the PR)

Reply:
You have a point! Will look through the PR and check